src/main/kotlin/no/nav/klage/oppgave/api/controller/BehandlingController.kt

```diff
@@ -0,0 +1,65 @@
+package no.nav.klage.oppgave.api.controller
+
+import io.swagger.annotations.Api
+import io.swagger.annotations.ApiParam
+import no.nav.klage.oppgave.api.view.BehandlingEditedView
+import no.nav.klage.oppgave.config.SecurityConfiguration.Companion.ISSUER_AAD
+import no.nav.klage.oppgave.repositories.InnloggetSaksbehandlerRepository
+import no.nav.klage.oppgave.service.BehandlingService
+import no.nav.klage.oppgave.util.getLogger
+import no.nav.klage.oppgave.util.logBehandlingMethodDetails
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.web.bind.annotation.*
+import java.util.*
+
+@RestController
+@Api(tags = ["kabal-api"])
+@ProtectedWithClaims(issuer = ISSUER_AAD)
+@RequestMapping("/klagebehandlinger")
+class BehandlingController(
+    private val behandlingService: BehandlingService,
+    private val innloggetSaksbehandlerRepository: InnloggetSaksbehandlerRepository,
+) {
+    companion object {
+        @Suppress("JAVA_CLASS_ON_COMPANION")
+        private val logger = getLogger(javaClass.enclosingClass)
+    }
+
+    @PostMapping("/{behandlingId}/sattpaavent")
```

Comment:
Kunne vært en PUT med true/false, hør med FE hva de ser for seg?

Reply:
Ja, skal høre med dem.